Create data_management_basics.md (#281)
arcus/education_modules: data_management_basics/data_management_basics.md

<!--
author:   Ene Belleh
email:    bellehe@chop.edu
version:  1.0.0
module_template_version: 3.0.0
language: en
narrator: US English Female
title: Research Data Management Basics
comment: Learn the basics about research data management.
long_description:  If you conduct research or work with research data or researchers, it's likely that research data management topics affect you.  Learn what research data management is, how to think about it in a structured way, and understand its scientific importance.
estimated_time: 40 minutes

@learning_objectives  

After completion of this module, learners will be able to:

- Define research data management

> - Explain the significance of data management as a component of the responsible conduct of research

Author: The phrase "responsible conduct of research" is a term of art in the sciences and I think most people will recognize it more easily in that kind of phrasing.  Still, I tried to simplify in the suggestion below!

```suggestion
- Explain why data management forms an important part of the responsible conduct of research
```

</div>
*****

## Research Data Lifecycle

At the start of this module, we gave a description of research data management that had two major aspects: work done to keep a research project going, and work done to make research data usable after a project ends.  

We're going to offer a more complex taxonomy of research data management here, based on RDM tasks that belong to each of the five stages in the **research data lifecycle**.  Before we share our visualization of the research data lifecycle, we'd like to offer a caveat.  

> All models are wrong, but some are useful." - George E. P. Box

You've perhaps seen visualizations of the research data lifecycle that show a circular diagram.  We'll add our own circular diagram to the mix, but keep in mind that you may see these stages called slightly different things in different places!  You may also see similar diagrams with more or fewer steps.  We'll go into each phase of research data management in greater detail.

> 

Author: great question and I'm not sure, because it's a generated diagram from mermaid!  Let me do a bit of research...

> 

Author: https://github.com/mermaid-js/mermaid/issues/2732 seems to imply that mermaid is increasingly acceptable but there is some back and forth on whether it actually works. I've just added an additional paragraph to explain what the diagram shows.

> 

Author: @Damizen2 I fixed this via an explainer paragraph below.

<div style = "background-color:white;">

<script style="display: block" run-once="true" modify="false">
mermaid.initialize({});

var svg = mermaid.render(

> 'approval_flowchart',

Reviewer: does this fall under the list of images that need an alt text?

Author: This isn't an image, actually, it's a diagram of shapes on the html5 canvas, so there's no attached image.  I added an explainer paragraph.

  D --> E[Long-Term\\nManagement]
  E --> A
`,
function(g) {
    return true;
})

"HTML: " + svg
</script>

</div>

### Discovery & Planning

This stage of the research data lifecycle involves creating formal and informal plans for how research will be conducted.  Much of this planning will result in official documentation (such as a data management and sharing plan, a research protocol, or study pre-registration).  This documentation itself will also form part of the research data that is managed.  

Researchers and research staff have to make a number of important decisions at this stage. Important research data management tasks for this stage of research include, but are not limited to:

* deciding whether a project will create new datasets, combine existing datasets, or analyze existing datasets
* identifying privacy, confidentiality, and other ethical concerns
* thinking about the format and content of documentation as well as the metadata standards to use when describing the data
* identifying potential project users
* choosing an appropriate data repository to store the project's data
* figuring out the costs associated with data management

### Data Collection

> At this point, researchers execute workflows and methods for arranging data, including digital data.  Digital data considerations include file backup plans and storage planning. This stage also includes exercising protocols for quality control and the establishment of security and access controls, whether for physical samples, digital files, or both.

Author: Agree, I'll take a stab at improving this!

> At this point, researchers execute workflows and methods for arranging data, including digital data.  Digital data considerations include file backup plans and storage planning. This stage also includes exercising protocols for quality control and the establishment of security and access controls, whether for physical samples, digital files, or both.

Author: See if you like my new text @Damizen2

> At this point, researchers execute workflows and methods for arranging data, including digital data.  Digital data considerations include file backup plans and storage planning. This stage also includes exercising protocols for quality control and the establishment of security and access controls, whether for physical samples, digital files, or both.

Author: ```suggestion
At this point, researchers carry out the workflows and methods they planned for their data organization.  This includes both physical assets as well as digital data.  For digital data, it is important to ensure that there are file backup plans and that appropriate storage is planned (including aspects such as the size, cost, and security of storage). At this point, investigators should be implementing their plans for quality control, security, and access controls, whether for physical samples, digital files, or both.
```

### Publication and Sharing

Most publishers have their own requirements related to data formats, documentation of scripts used to clean and analyze data, and other materials that must be submitted along with a manuscript.  During the publication stage, researchers should consider their documentation carefully.  Are the submitted materials sufficient to permit the data to be used effectively again by a third party?  To allow reviewers to examine their methods and statistical rigor?  If primary data cannot be shared for ethical reasons, sample fabricated data can be used to demonstrate the expected data format used by scripts and models used in the study.

### Long-term Management

During this phase, researchers publish their data and findings, turn in reports, and deposit data and supporting materials into an archive or repository.  This is a highly contextual part of the research data management lifecycle, which will change according to the norms of various disciplines.

<div class = "history">
<b style="color: rgb(var(--color-highlight));">Historical context</b><br>

> The United States government, which funds many research efforts, has been at the forefront of data repository creation.  Government agencies have also developed several policies and glossaries to help researchers be better stewards of federal funds and organize their data in ways that are better for the conduct of science.  The 2010 [Administrative Order 212-15 of the National Oceanic and Atmospheric Association](https://nosc.noaa.gov/EDMC/nao_212-15.php), for example, gave definitions of **Data Stewards**, **Data Management**, **Data Management Services**, and **Data Stewardship**.

Author: There's no image here?  Or do you mean the icon?  The icon is a font element and doesn't need alt text, it's purely decorative

[[X]] Funding agencies supporting the research require research data management.
[[ ]] Research data management has recently appeared as a new practice, and performing this work shows that researchers are up-to-date.
[[?]] There are three correct answers!
****
<div class = "answer">

Research data management is important for multiple reasons, among which we highlighted three in this question:

* Research data management helps researchers create higher quality research output.
* Research data management supports the long-term preservation of data for future use.
* Funding agencies supporting the research require research data management.

It's false that publishers don't apply their own standards.  Publishers have an important role to play in setting high standards for researchers who submit manuscripts for consideration.  It's also false that research data management is a new practice.  While it's true that RDM has gotten much more attention lately, research data management has been taking place since the first researchers developed the scientific method, and it's changed over time to adapt to new circumstances and requirements.

</div>
***

## Additional Resources

> The [data sharing policies and helpful information](https://sharing.nih.gov/data-management-and-sharing-policy

Reviewer: i think the code here is a little faulty. When I went into the liascript version of the code, the full link was visible and not hyperlinked

Author: Whoops I'll fix
